modules/agar/src/main/ts/ephox/agar/api/FocusTools.ts

```diff
@@ -56,25 +56,25 @@ const sTryOnSelector = <T>(label: string, doc: Element<any>, selector: string):
     )
   );
 
-const cSetFocus = (label: string, selector: string): Chain<Element, Element> => {
+const cSetFocus = (label: string, selector: string): Chain<Element, Element> =>
   // Input: container
-  return Chain.fromChains([
+  Chain.fromChains([
     Chain.control<Element, Element, Element>(
       UiFinder.cFindIn(selector),
       Guard.addLogging(label)
     ),
     cSetFocused
-  ]);
-};
+  ])
+;
```

Comment:
The semi colons in this file look to be in the incorrect place?

Reply:
All these should be fixed now.